### ISSUE TITLE
import problems utilits register_custom_env_task and config import load_config_dict

### ISSUE THREAD

**Issue by rsarpongstreetor** [facebookresearch/BenchMARL#164]
from benchmarl.utils.env_utils import register_custom_env_task # Import from benchmarl.utils.env_utils
from benchmarl.utils.config import load_config_dict

Please I am having problem in running these two import lines

**Comment by matteobettini** [comment]
Benchmarl does not have those components, where did you get this from?

**Comment by rsarpongstreetor** [comment]
Colab suggested  them whilst i was implementing extension in custom environment

**Comment by rsarpongstreetor** [comment]
rom benchmarl.experiment import Experiment, ExperimentConfig
from benchmarl.algorithms import MappoConfig, MasacConfig, QmixConfig, IppoConfig
from benchmarl.benchmark import Benchmark
from benchmarl.environments import VmasTask  # Import VmasTask directly
from benchmarl.experiment import ExperimentConfig
from benchmarl.models.mlp import MlpConfig
from benchmarl.models.gnn import GnnConfig
import torch_geometric.nn.conv
import torch.nn as nn
from benchmarl.models.common import SequenceModelConfig, ModelConfig
import benchmarl.environments

# Create an empty __init__.py file in benchmarl/utils
!touch /content/BenchMARL/benchmarl/utils/__init__.py

# Now your imports should work


# Define and register your custom environment task
@register_custom_env_task(name="task_1", env_id="ANFUELEnv")
class CustomEnvTask(VmasTask):
    def __init__(self, **kwargs):
        super().__init__(**kwargs)

# ... (rest of your code)

config = ExperimentConfig.get_from_yaml()
config.save_folder = "/content/drive/MyDrive/Gcnresultenvj25"

# Load your custom task configuration
task_config_path = "/content/BenchMARL/benchmarl/conf/task/custom/Task_1.yaml" # Adjusted Path
task_config_dict = load_config_dict(task_config_path)

tasks = [CustomEnvTask.task_1]  # Assuming CustomEnvTask is defined and task_1 is an attribute

# Iterate through the tasks and run the experiment for each
for task in tasks:
    # Create the experiment with IPPO and a GNN model
    experiment = Experiment(
        algorithm_config=IppoConfig.get_from_yaml(),  # Use IPPO algorithm
        model_config=GnnConfig(  # Use a GNN model
            topology="full",
            self_loops=False,
            position_key=None,
            pos_features=0,
            exclude_pos_from_node_features=False,
            gnn_class=torch_geometric.nn.conv.GATv2Conv,
            gnn_kwargs={},
        ),
        critic_model_config=SequenceModelConfig(  # Use a sequence model for the critic
            model_configs=[
                MlpConfig(num_cells=[8], activation_class=nn.Tanh, layer_class=nn.Linear),
                GnnConfig(
                    topology="full",
                    self_loops=False,
                    gnn_class=torch_geometric.nn.conv.GraphConv,
                ),
                MlpConfig(num_cells=[6], activation_class=nn.Tanh, layer_class=nn.Linear),
            ],
            intermediate_sizes=[5, 3],
        ),
        seed=0,
        config=config,  # Pass the configured ExperimentConfig object
        task=task,  # Pass the current task from the list
    )

    # Run the experiment for the current task
experiment.run()

# ... (continue with your experiment setup)
